cocos/2d/CCScene.cpp

```diff
@@ -167,7 +172,7 @@ void Scene::render(Renderer* renderer)
         //clear background with max depth
         camera->clearBackground(1.0);
         //visit the scene
-        visit(renderer, transform, 0);
+        visit(renderer, transform, Node::FLAGS_TRANSFORM_DIRTY);
```

Comment:
yes, let's find the real bug... but please, don't merge this PR with the DIRTY_FLAG on. Thanks.